app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js

```diff
@@ -63,6 +63,15 @@ define([
                 isFileUploaded = false,
                 self = this;
 
+            $('[data-action="add-to-wishlist"]').on('click', function () {
+                var validator = $('#product_addtocart_form').validate();
```

Comment:
Please define the selector in the configuration block. 
Also, all variable declaration should be on the top of the function.